neo4j/README.md

```diff
@@ -0,0 +1,59 @@
+<!--
+Copyright (c) 2012 - 2016 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+## Quick Start
+
+This section describes how to run YCSB on Neo4J running locally.
+
+### 1. Set Up YCSB
+
+First, clone the YCSB git repository:
+
+    git clone https://github.com/brianfrankcooper/YCSB.git
+    cd YCSB
+
+You can now compile the project:
+
+    mvn clean package
+
+If you want to only compile the Neo4J binding, use:
+
+    mvn -pl com.yahoo.ycsb:neo4j-binding -am clean package
+
+### 2. DB creation with Neo4J
```

Comment:
Can this point to an existing Neo4j server/cluster? Does it always have to create a neo4j.db?